### PR TITLE
Fix sd3-lite clip-l model swapped outputs

### PR DESCRIPTION
**Pull request by ikalinic**
The clip-l model from [stable-diffusion-3-lite-onnx](https://huggingface.co/TensorStack/stable-diffusion-3-lite-onnx) has swapped outputs, so this results in runtime error because of mismatched shapes. This fix proposes swapping the output tensors for the clip-l outputs if the shape doesn't match the required shape for one of the outputs, making stable-diffusion-3-lite-onnx run successfully.

### CODE REVIEW

File: examples/diffusion/python_stable_diffusion_3/txt2img.py [hunk 1]
```diff
@@ -485,6 +485,11 @@ def encode_token_weights(self, model_name, token_weight_pairs):
     def get_embeddings(self, prompt_tokens):
         l_out, l_pooled = self.encode_token_weights("clip-l",
                                                     prompt_tokens["l"])
+        if l_out.shape != (1, 77, 768):
```
Comment (kahmed10) [inline examples/diffusion/python_stable_diffusion_3/txt2img.py:488]: let's add a comment here explaining why this change is needed

**Review by kahmed10 — APPROVED** [review]

**Comment by causten** [comment]
Just fix the format problem.  Ignore the license problem since you didnt touch that code
